<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="9" hostname="Hasans-MacBook-Pro.local" name="com.kff.test.FieldsTests" tests="11" failures="0" timestamp="21 Aug 2018 22:31:11 GMT" time="5.331" errors="2">
  <testcase name="ClearSubmit" time="0.465" classname="com.kff.test.FieldsTests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.kff.test.FieldsTests.ClearSubmit(FieldsTests.java:163)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- ClearSubmit -->
  <testcase name="SelectState" time="0.403" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- SelectState -->
  <testcase name="tearDown" time="1.039" classname="com.kff.test.FieldsTests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.kff.test.TestBase.tearDown(TestBase.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- tearDown -->
  <testcase name="SelectStateNegative" time="0.401" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- SelectStateNegative -->
  <testcase name="enterIncomeFunctionality" time="0.457" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- enterIncomeFunctionality -->
  <testcase name="householdIncomeNegative" time="0.628" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- householdIncomeNegative -->
  <testcase name="householdIncomeFieldEmpty" time="0.436" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- householdIncomeFieldEmpty -->
  <testcase name="householdIncomeNegativeEmpty" time="0.432" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- householdIncomeNegativeEmpty -->
  <testcase name="isCoverageSelectNo" time="0.475" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- isCoverageSelectNo -->
  <testcase name="NumberofChildren" time="0.595" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- NumberofChildren -->
  <testcase name="clearButtons" time="0.000" classname="com.kff.test.FieldsTests">
    <skipped/>
  </testcase> <!-- clearButtons -->
</testsuite> <!-- com.kff.test.FieldsTests -->
